### DISCUSSION TITLE
How to add global background color ?

### DISCUSSION THREAD

**Discussion by asenseofpradhyu** [chakra-ui/chakra-ui#3948] [Q&A]
Can anyone help me to add global background color with example..

**Comment by chr-ge** [comment]
Take a look at the [global-styles](https://chakra-ui.com/docs/features/global-styles) section of the docs.

If you want to add a global bg color:
```jsx
const theme = extendTheme({
  styles = {
    global: {
      body: {
        bg: 'gray.800',
      },
    },
  },
})
```

If you want to have a different bg color for light and dark mode:
```jsx
import { mode } from "@chakra-ui/theme-tools"

const theme = extendTheme({
  styles = {
    global: (props) => ({
      body: {
        bg: mode('white', 'gray.800')(props),
      },
    }),
  },
})
```

**Reply by asenseofpradhyu** [reply]
Hello @chr-ge Thank you for your answer can i use Hexcode like this ?
`const theme = extendTheme({
  styles = {
    global: {
      body: {
        bg: '#E5E5E5,
      },
    },
  },
})`

**Reply by chr-ge** [reply]
Yes, you can.
